.github/workflows/deploy.yml

```diff
@@ -1,44 +1,126 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Deploy Demo App Workflow
+name: Deploy Meeting And Meeting Readiness Demos
```

Comment:
nit: Can we keep the previous name as we might add more demos in the future :)?

Reply:
fixed